components/tikv_alloc/src/jemalloc.rs

```diff
@@ -89,6 +92,26 @@ mod profiling {
         }
     }
 
+    pub fn activate_prof() -> ProfResult<()> {
+        unsafe {
+            if let Err(e) = jemallocator::mallctl_set(PROF_ACTIVE, true) {
+                error!("failed to activate profiling: {}", e);
+                return Err(ProfError::JemallocError(e));
+            };
+        }
+        return Ok(());
```

Comment:
```suggestion
        Ok(())
```

`return` is not needed for the final expression.